### ISSUE TITLE
`--print` breaks recursive `edit`

### ISSUE THREAD

**Issue by jvacek** [Keeper-Security/Commander#203]
When i run any rotation with the `--print` flag, e.g. `r <UID> --print` I get the following error.
```python
'str' object has no attribute 'decode'
```
If I'm using `--match` to rotate multiple records, e.g. `r --match <pattern> --print`the rotation will complete the first one successfully and then stop after giving the error above, not progressing to the following ones.

Here are some specs for troubleshooting, hope that helps a bit

```
╰─$ python3 --version
Python 3.7.3

╰─$ keeper --version
Keeper Commander, version 4.21

╰─$ pip3 --version
pip 18.1 from /usr/lib/python3/dist-packages/pip (python 3.7)

╰─$ neofetch 
       _,met$$$$$gg.          jvacek@xxx 
    ,g$$$$$$$$$$$$$$$P.       ------------------ 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 10 (buster) x86_64 
 ,$$P'              `$$$.     Host: Latitude 5401 
',$$P       ,ggs.     `$$b:   Kernel: 5.3.0-0.bpo.2-amd64 
```

**Comment by sk-keeper** [comment]
Thank you for reporting this issue. The fix is checked in to Commander's repository.
This change will be released in version 4.24 soon.

**Comment by sk-keeper** [comment]
The Commander release with the fix is published
